### PR TITLE
update syslog docs to include transaction.detail.requestApiTokenId info

### PR DESCRIPTION
**Pull request by jackwang-okta**
## Description:
- **What's changed?** Updated syslog docs to include introduced `transaction.detail.requestApiTokenId` info
- **Is this PR related to a Monolith release?** 2022.02.1
### Resolves:

* [OKTA-465398](https://oktainc.atlassian.net/browse/OKTA-465398)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/system-log/index.md [hunk 1]
```diff
@@ -339,6 +339,19 @@ For example, a Transaction object such as the following:
 
 indicates that a `WEB` request with `id` `Wn4f-0RQ8D8lTSLkAmkKdQAADqo` has created this event.
 
+A Transaction object such as the following:
```
Comment (ericzhou-okta) [inline packages/@okta/vuepress-site/docs/reference/api/system-log/index.md:342]:
```suggestion
A Transaction object such as the following with a `requestApiTokenId` in the `detail` object, for example,
```

**Comment by ericzhou-okta** [comment]
@dandennhardt-okta @brianduffield-okta Do you guys think it would be valuable to add this information (or link it) from the API token docs page, or can we skip that?

**Comment by dandennhardt-okta** [comment]
@ericzhou-okta - Yeah, good call. I could see something like this on the `See also` section of [this page](https://developer.okta.com/docs/guides/create-an-api-token/main/) :

> See <link in syslog page to transaction object> for information on seeing the API token id in the Okta System Log.

**Review by brianduffield-okta — COMMENTED** [review]
Few change requests

**Review by brianduffield-okta — APPROVED** [review]
LGTM

**Review by ericzhou-okta — APPROVED** [review]

**Pull request merged**